packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -531,4 +534,30 @@ export class Telemetry {
 
     return exceptionData;
   }
+
+  static traverseCodedErrorStruct(
```

Comment:
This is close, but could you instead perhaps call this something like `sanitizeAny` that can take `any` type, then if it's an Array, recurse through the items or if it's an object, recurse through the fields?